docs/quick_start_en.md

```diff
@@ -173,28 +190,20 @@ Verify that the file is properly uploaded by pointing your browser to the [VOSpa
 
 Now you want to save the VM with your software installed. Return to your browser on the OpenStack dashboard.
 
-* Save the state of your VM by navigating to the **Instances** window of the dashboard. and click on the **Create Snapshot** button to the right of your VM instance's name. After selecting a name for the snapshot of the VM instance, e.g., ```quick_start-0.1```, click the **Create Snapshot** button. It will eventually be saved and listed in the VM **Images** window, and will be available next time you launch an instance of that VM image.
+* Save the state of your VM by navigating to the **Instances** window of the dashboard. and click on the **Create Snapshot** button to the right of your VM instance's name. After selecting a meaningful name for the snapshot of the VM instance, e.g., ```image-reduction-2021-06-26```, click the **Create Snapshot** button. The VM instance will eventually be saved and listed in the VM **Images** window, and will be available next time you launch an instance of that VM image in a few minutes.
 
 While the system is taking a snapshot of your VM, avoid doing anything on your VM.
 
-```IMPORTANT``` : if you do not create a snapshot of the VM, your work of this VM will be lost if the VM is deleted. Volume based VMs will keep your work, but Volume based VM can not be used for batch processing.
+```IMPORTANT``` : if you do not create a snapshot of the VM, your work of this VM will be lost if the VM is deleted. Volume-based VMs do not need snapshots and will persist the files directly on a persistent disk, but Volume-based VM can not be used for batch processing.
```

Comment:
" ... if you do not create a snapshot of the VM, you will lose all of your work when the VM is deleted. ... "